json_serializable/lib/src/json_serializable_generator.dart

```diff
@@ -558,3 +583,28 @@ InvalidGenerationSourceError _createInvalidGenerationError(
   return new InvalidGenerationSourceError(message,
       todo: 'Make sure all of the types are serializable.');
 }
+
+/// Returns a list of all instance, [FieldElement] items for [element] and
+/// super classes.
+List<FieldElement> _getFields(ClassElement element) {
```

Comment:
[optional] I have a personal vendetta against methods named `get*`. Maybe `findFields` or `allFields`?

Reply:
Changed to `_listFields`